app/routes/lending/LendableObjectsAdmin.tsx

```diff
@@ -0,0 +1,159 @@
+import moment from 'moment-timezone';
+import { Helmet } from 'react-helmet-async';
+import { Link } from 'react-router-dom';
+import { Content } from 'app/components/Content';
+import NavigationTab from 'app/components/NavigationTab';
+import styles from './LendableObjectsAdmin.css';
+import { useState } from 'react';
+import { ListLendableObject } from 'app/store/models/LendableObject';
+import { LendingRequest, status } from './components/LendingRequest';
+
+const LendableObjectsAdmin = () => {
+  const lendingRequests = [
+    {
+      id: 1,
+      user: 'Test Testesen',
+      startTime: moment().subtract({ hours: 2 }),
+      endTime: moment(),
+      message: 'Jeg vil gjerne låne Soundboks til hyttetur:)',
+      status: status.PENDING,
+      lendableObject: {
+        id: 1,
+        title: 'Grill',
+        image: 'https://food.unl.edu/newsletters/images/grilled-kabobs.jpg',
+      },
+    },
+    {
+      id: 2,
+      user: 'Test Testesen',
+      startTime: moment().subtract({ hours: 2 }),
+      endTime: moment(),
+      message: 'Jeg vil gjerne låne Soundboks til hyttetur:)',
+      approved: false,
+      status: status.PENDING,
+      lendableObject: {
+        id: 2,
+        title: 'Grill',
+        image: 'https://food.unl.edu/newsletters/images/grilled-kabobs.jpg',
+      },
+    },
+    {
+      id: 3,
+      user: 'Test Testesen',
+      startTime: moment().subtract({ hours: 2 }),
+      endTime: moment(),
+      message: 'Jeg vil gjerne låne Soundboks til hyttetur:)',
+      approved: false,
+      status: status.APPROVED,
+      lendableObject: {
+        id: 2,
+        title: 'Grill',
+        image: 'https://food.unl.edu/newsletters/images/grilled-kabobs.jpg',
+      },
+    },
+    {
+      id: 4,
+      user: 'Test Testesen',
+      startTime: moment().subtract({ hours: 2 }),
+      endTime: moment(),
+      message: 'Jeg vil gjerne låne Soundboks til hyttetur:)',
+      approved: false,
+      status: status.DENIED,
+      lendableObject: {
+        id: 2,
+        title: 'Grill',
+        image: 'https://food.unl.edu/newsletters/images/grilled-kabobs.jpg',
+      },
+    },
+  ];
+
+  const lendableObjects: Array<ListLendableObject> = [
+    {
+      id: 1,
+      title: 'Grill',
+      image: 'https://food.unl.edu/newsletters/images/grilled-kabobs.jpg',
+    },
+    {
+      id: 2,
+      title: 'Soundboks',
+      image: 'https://food.unl.edu/newsletters/images/grilled-kabobs.jpg',
+    },
+    {
+      id: 3,
+      title: 'Soundboks2',
+      image: 'https://food.unl.edu/newsletters/images/grilled-kabobs.jpg',
+    },
+    {
+      id: 4,
+      title: 'Prinsessekjole',
+      image: 'https://food.unl.edu/newsletters/images/grilled-kabobs.jpg',
+    },
+    {
+      id: 5,
+      title: 'Falk',
+      image: 'https://food.unl.edu/newsletters/images/grilled-kabobs.jpg',
+    },
+  ];
+
+  const [showFetchMore, setShowFetchMore] = useState(true);
+
+  return (
+    <Content>
+      <Helmet title="Utlån" />
```

Comment:
A different helmet would be nice here to actually indicate that you're in the admin page